cherrypy/test/test_config.py

```diff
@@ -221,8 +221,8 @@ def testHandlerToolConfigOverride(self):
         # the favicon in the page handler to be '../favicon.ico',
         # but then overrode it in config to be './static/dirback.jpg'.
         self.getPage('/favicon.ico')
-        self.assertBody(open(os.path.join(localDir, 'static/dirback.jpg'),
-                             'rb').read())
+        with open(os.path.join(localDir, 'static/dirback.jpg'), 'rb') as tf:
+            self.assertBody(tf.read())
```

Comment:
It'd probably be cleaner with `pathlib` API. But it doesn't matter now. Just something to keep in mind for the future.